src/OpenLoco/Widget.h

```diff
@@ -40,6 +40,10 @@ namespace OpenLoco::Ui
         static void drawViewportCentreButton(Gfx::Context* context, const Window* window, const WidgetIndex_t widgetIndex);
         static void drawTab(Window* w, Gfx::Context* ctx, int32_t imageId, WidgetIndex_t index);
 
+        // typical tab width, to be used in most (all?) cases
+        static constexpr uint16_t kDefaultTabWidth = 30;
+        static void leftAlignTabs(Window& window, uint8_t firstTabIndex, uint8_t lastTabIndex, uint16_t tabWidth);
```

Comment:
We might want to make `kDefaultTabWidth` the actual default, so we don't have to pass it in every call, e.g.:
```suggestion
        static constexpr uint16_t kDefaultTabWidth = 30;
        static void leftAlignTabs(Window& window, uint8_t firstTabIndex, uint8_t lastTabIndex, uint16_t tabWidth = kDefaultTabWidth);
```

Reply:
Did consider this, and am fine with either- the counterpoint as suggested by LeftOfZen is that having to pass the default makes the behavior more explicit and clear.

Reply:
yeah I initially suggested not defaulting the param when we were discussing it, but it is used quite a lot here so defaulting it does make sense as well. I don't mind either way if we default it or not